app/services/analytics.rb

```diff
@@ -58,6 +58,7 @@ def uuid
   PHONE_CHANGE_REQUESTED = 'Phone Number Change: requested'.freeze
   PROFILE_ENCRYPTION_INVALID = 'Profile Encryption: Invalid'.freeze
   PROFILE_RECOVERY_CODE_CREATE = 'Profile: Created new recovery code'.freeze
+  RESEND_EMAIL_CONFIRMATION = 'Email Confirmation requested due to invalid token'.freeze
```

Comment:
If we named it `EMAIL_CONFIRMATION_RESEND` we could alphabetize it next to the existing `EMAIL_CONFIRMATION` key? (just an idea)